### PR TITLE
Adds rln as a submodule

### PR DESCRIPTION
**Pull request by staheri14**
Addresses https://github.com/status-im/nim-waku/issues/401

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -106,3 +106,7 @@
 [submodule "vendor/nim-web3"]
 	path = vendor/nim-web3
 	url = https://github.com/status-im/nim-web3.git
+[submodule "vendor/rln"]
+	path = vendor/rln
+	url = https://github.com/kilic/rln
+	branch = full-node
```
Comment (oskarth) [inline .gitmodules:112]: Curious, what is this branch for? Vs master

**Review by oskarth — APPROVED** [review]
LGTM!

**Review by jm-clius — APPROVED** [review]
Great, thanks! This will simplify things.

**Comment by staheri14** [comment]
> Great, thanks! This will simplify things.

Sure! glad it helps! :)

File: .gitmodules (hunk 1 above)
Comment (staheri14) [reply .gitmodules:112]: @oskarth Good question! this is a feature branch that most of the things we need are under this branch, as far as I checked with Onur, full-node branch will be ultimately merged with the master

**Pull request merged**